### PR TITLE
Update remove-paywall extension

### PR DESCRIPTION
**Pull request by artivilla**
## Description

Handling this issue: https://github.com/raycast/extensions/issues/11019

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [ ] I checked that files in the `assets` folder are used by the extension itself
- [ ] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/remove-paywall/src/remove-paywall.tsx [hunk 1]
```diff
@@ -18,6 +18,10 @@ export default async (props: LaunchProps<{ arguments: Arguments.RemovePaywall }>
       throw url;
     }
 
+    if (service === "https//freedium.cfd") {
+      open(service.concat(`/${url}`));
+    }
+
```
Comment (DannySeidel) [inline extensions/remove-paywall/src/remove-paywall.tsx:24]: Hey @artivilla,

First of all, thank you for your contribution. This is not required as the matching webpage is already open in line 26 for the chosen service.
```suggestion
```

Comment (artivilla) [reply extensions/remove-paywall/src/remove-paywall.tsx:24]: oh right. for some reason I wrongly assumed at the time of writing it had its own format.

File: extensions/remove-paywall/src/remove-paywall.tsx [hunk 2]
```diff
@@ -1,8 +1,12 @@
 import { Toast, getPreferenceValues, LaunchProps, open, showToast } from "@raycast/api";
 import { getUrl } from "./utils";
 
+interface Preferences {
+  service: string;
+}
+
```
[the remark this answers is not in the document]
Comment (artivilla) [reply extensions/remove-paywall/src/remove-paywall.tsx:7]: oh interesting. i've seen it defined in a number of ext and my [quick lint js](https://marketplace.visualstudio.com/items?itemName=quick-lint.quick-lint-js) calls it out. will remove.

File: extensions/remove-paywall/package.json [hunk 3]
```diff
@@ -68,12 +73,16 @@
         {
           "title": "RemovePaywall (removepaywall.com)",
           "value": "https://www.removepaywall.com"
+        },
+        {
+          "title": "Freedium.cfd (freedium.cfd)",
```
Comment (tegola) [inline extensions/remove-paywall/package.json:78]: Just nitpicking here: if the service name is "Freedium", omit the `.cfd` suffix in its name

```suggestion
          "title": "Freedium (freedium.cfd)",
```

While you're there, can you also make 12ft consistent with the naming?

```
- "title": "12ft.io",
+ "title": "12ft (12ft.io)",
```

Thanks!

File: extensions/remove-paywall/CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,9 @@
 # Remove Paywall Changelog
 
+## [Added freedium] - 2024-02-28
+
+- Added support for freedium service
```
Comment (tegola) [inline extensions/remove-paywall/CHANGELOG.md:5]:
```suggestion
- Added support for Freedium
```

**Review by tegola — CHANGES_REQUESTED** [review]

**Comment by DannySeidel** [comment]
Hey @pernielsentikaer,

I think the PR ready for review now 🔥

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**